File: client-core/src/main/java/hivens/core/data/ServerListResponse.java
package hivens.core.data;

import com.google.gson.annotations.SerializedName;
import java.util.List;

/**
 * Модель данных (DTO) для ответа API, содержащего список серверов.
 * (Аналог ba.class).
 */
public record ServerListResponse(
    
    // (Имя поля 'servers' - это предположение, 
    // основанное на n.java -> this.a.a.forEach(...))
    @SerializedName("servers") 
    List<ServerData> servers
    
    // (Могут быть и другие поля, например, 'news')
) {}

File: client-core/src/main/java/hivens/core/data/ServerListResponse.kt
package hivens.core.data

import com.google.gson.annotations.SerializedName

/**
 * Модель данных (DTO) для ответа API, содержащего список серверов.
 */
data class ServerListResponse(
    @SerializedName("servers")
    val servers: List<ServerData> = emptyList()
)